packages/requisitions/src/SupplierRequisition/DetailView/api.ts

```diff
@@ -39,6 +49,43 @@ const linesGuard = (
   throw new Error('Unknown');
 };
 
+const createUpdateSupplierRequisitionInput = (
```

Comment:
In general this is just a copy paste of the same thing from the inbound api replacing 'InboundShipment' with 'SupplierRequisition' 😁